spec/factories/legacy_hearing.rb

```diff
@@ -48,12 +48,8 @@
       )
     end
 
-    after(:create) do |hearing, evaluator|
-      hearing.update(
-        scheduled_in_timezone: Constants::REGIONAL_OFFICE_INFORMATION.dig(
-          evaluator.regional_office, "timezone"
-        ) || "America/New_York"
-      )
+    after(:create) do |hearing|
+      hearing.update(scheduled_in_timezone: nil)
     end
```

Comment:
```suggestion
scheduled_in_timezone { nil }
```
This would allow us to specify a custom `scheduled_in_timezone` and override the `nil` value in tests moving forward.